bp_common/test/src/perch/bp_utils.h

```diff
@@ -18,4 +18,43 @@ void bp_finish(uint8_t code);
 
 #define BP_CFG_BASE_ADDR ((char *)(0x00200000))
 
```

Comment:
This might be nitty, but can we put the accelerator specific stuff in a vdp.h instead of the main perch?